index.js

```diff
@@ -308,6 +320,9 @@ Router.prototype.lookup = function lookup (req, res) {
 }
 
 Router.prototype.find = function find (method, path, version) {
+  if (!this.caseSensitive) {
```

Comment:
`if (this.caseSensitive === false)?`
It should make V8 happier (but I'm not super sure because the value is already a Boolean).

Reply:
updated.